File: 20190903-ModelTest/20190903-ModelTest/ViewController.m
//
//  ViewController.m
//  20190903-ModelTest
//
//  Created by dongouc on 2019/9/3.
//  Copyright © 2019 dongouc. All rights reserved.
//

#import "ViewController.h"

//#import <dyfOAtestSDK/testForVc.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
//    testForVc *testVc = testForVc.new;
//    [testVc test001];
    Class targetClass = NSClassFromString(@"testForVc");
    SEL sel = NSSelectorFromString(@"test001");
    [targetClass.new performSelector:sel];
}


@end

File: my_flutter/20191010-001/20191010-001/ViewController.swift
//
//  ViewController.swift
//  20191010-001
//
//  Created by dongouc on 2019/10/10.
//  Copyright © 2019 dongouc. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        view.backgroundColor = UIColor.red
        
        let btn = UIButton()
        btn.backgroundColor = UIColor.white
        btn.frame = CGRect(x: 100, y: 100, width: 100, height: 30)
        btn.setTitle("tapToFlutter", for: .normal)
        btn.setTitleColor(UIColor.blue, for: .normal)
        btn.addTarget(self, action: #selector(_tapToFlutter(_:)), for: .touchUpInside)
        view.addSubview(btn)
    }


    @objc func _tapToFlutter(_ :UIButton) {
        //
        let vc = WebViewController()
        vc.modalPresentationStyle = .formSheet
        self.present(vc, animated: true, completion: nil)
        
    }
}

